tsconfig.json

```diff
@@ -12,6 +12,9 @@
     "noUnusedLocals": true,
     "strict": true
   },
+  "include":[
+    "js-src/graphql"
```

Comment:
Do you know why we need to explicitly include this folder? I wonder if we can fix it some other way by making sure that all the code under js-src is included.

Reply:
Removed, copied this from an early branch, its not needed.

Reply:
:+1: